android/src/main/java/io/parity/signer/components/IdentIcon.kt

```diff
@@ -16,6 +18,12 @@ import io.parity.signer.models.intoImageBitmap
 @Composable
 fun IdentIcon(identicon: List<UByte>, size: Dp = 28.dp) {
 	Image(
-		identicon.intoImageBitmap(), stringResource(R.string.description_identicon), modifier = Modifier.size(size)
+		identicon.intoImageBitmap(),
+		stringResource(R.string.description_identicon),
+		modifier = Modifier
+			.size(size)
+			.clip(
```

Comment:
I'm not sure, as I don't know Compose that much, but I think we should use something like this:
```suggestion
			.clip(CircleShape)
```
to keep it a proper circle no matter the size

Reply:
done